### PR TITLE
Allow multiple inclusion of module

### PR DESCRIPTION
**Pull request by EisenMax**

### CODE REVIEW

**Comment by EisenMax** [comment]
Da wäre die Version 2.1.2.

File: spec/rails_state_machine/model_spec.rb [hunk 1]
```diff
@@ -45,4 +45,17 @@ class MyClass < ActiveRecord::Base
 
   end
 
+  it 'does not remove already defined state machines if you include the module again afterwards (BUGFIX)' do
+    class_with_machine = Class.new(ActiveRecord::Base) do
+      include RailsStateMachine::Model
+      state_machine(:foo) {}
+      state_machine(:bar) {}
+      include RailsStateMachine::Model
```
Comment (foobear) [inline spec/rails_state_machine/model_spec.rb:53]: From my understanding, the issue which pointed to this bug was in a project where `include` happened before each state machine definition. Right now, the test does not reflect that and the bug might be reintroduced if only `state_machine` resets the `state_machines` registry, not the `include` statement itself.

IMO this is more robust and more explicit:

```suggestion
      include RailsStateMachine::Model
      state_machine(:foo) {}
      include RailsStateMachine::Model
      state_machine(:bar) {}
```

**Review by foobear — COMMENTED** [review]
The change itself looks good, thank you. I think we can improve the test a bit.

Please also update `CHANGELOG.md` in the "Unreleased" section.

We will increment the gem's version in a separate commit where we will also move contents from "Unreleased" into a section for the new version.

File: spec/rails_state_machine/model_spec.rb (hunk 1 above)
Comment (foobear) [reply spec/rails_state_machine/model_spec.rb]: Yes, the current bugged behavior is that `include` breaks it. However, the test itself somewhat depends on that while it should not.

Thanks for adjusting.

**Review by foobear — APPROVED** [review]

**Review by foobear — CHANGES_REQUESTED** [review]
Please also update CHANGELOG.md in the "Unreleased" section.

**Review by foobear — CHANGES_REQUESTED** [review]
Please use the `Unreleased` section in CHANGELOG.md and don't update the `version.rb` yet.

We currently have different unreleased changes and they should not try to assume a version number yet.

Version and changelog will be adjusted when we actually release.

**Review by foobear — APPROVED** [review]

**Pull request merged**